{"name": "Проверка получения информации по некорректному ID", "status": "broken", "statusDetails": {"message": "TypeError: PetApi.__init__() missing 1 required positional argument: 'logger'", "trace": "self = <tests.test_pet_api.TestPetApi object at 0x10289f9d0>, id = '0000000'\n\n    @allure.title('Проверка получения информации по некорректному ID')\n    @pytest.mark.parametrize('id', ['922337203685477580e7', '0000000', '111111111111111111111'])\n    def test_get_pet_by_non_existent_id(self, id):\n>       pet_api = PetApi()\nE       TypeError: PetApi.__init__() missing 1 required positional argument: 'logger'\n\nid         = '0000000'\nself       = <tests.test_pet_api.TestPetApi object at 0x10289f9d0>\n\ntests/test_pet_api.py:53: TypeError"}, "parameters": [{"name": "id", "value": "'0000000'"}], "start": 1714327507451, "stop": 1714327507451, "uuid": "af95f232-c197-4140-9129-43176da1cd5b", "historyId": "1eb5136321db88cbd12ede3b1420bcd8", "testCaseId": "acf0b1f06f555e3a99915b0d677338d3", "fullName": "tests.test_pet_api.TestPetApi#test_get_pet_by_non_existent_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_pet_api"}, {"name": "subSuite", "value": "TestPetApi"}, {"name": "host", "value": "d45763e349a4"}, {"name": "thread", "value": "97822-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_pet_api"}]}